silkworm.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	]
+}
```

Comment:
Please add .code-workspace to .gitignore

Reply:
OK